regression/goto-instrument/nondet-static-exclude_subfolder/package1/file.c

```diff
@@ -0,0 +1,10 @@
+#include "assert.h"
+#include "stdint.h"
+static uint32_t myVal = 3;
+static uint32_t myVal2 = 4;
```

Comment:
Is the use of `uint32_t` relevant for this test? Wouldn't `unsigned` or even `int` do?